docs/online-boutique-asm-manifests/kustomization.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Sample to illustrate how to customize the OnlineBoutique deployments, you need to replace:
+# ONLINEBOUTIQUE_NAMESPACE in this current kustomization.yaml
+# ONLINEBOUTIQUE_NAMESPACE in base/for-namespace/kustomization.yaml
+# ONLINEBOUTIQUE_NAMESPACE and ASM_VERSION in base/for-asm-channel/kustomization.yaml
+# ONLINEBOUTIQUE_NAMESPACE in authorization-policies/for-namespace/kustomization.yaml
+# ONLINEBOUTIQUE_NAMESPACE, INGRESS_GATEWAY_NAMESPACE and INGRESS_GATEWAY_NAME in authorization-policies/for-ingress-gateway/kustomization.yaml
+# ONLINEBOUTIQUE_NAMESPACE in sidecars/for-namespace/kustomization.yaml
+# before deploying this file.
+# As an example:
+# sed -i "s/ONLINEBOUTIQUE_NAMESPACE/onlineboutique/g" kustomization.yaml
+# sed -i "s/ONLINEBOUTIQUE_NAMESPACE/onlineboutique/g" base/for-namespace/kustomization.yaml
+# sed -i "s/ONLINEBOUTIQUE_NAMESPACE/onlineboutique/g;s/ASM_VERSION/asm-managed-rapid/g" base/for-asm-channel/kustomization.yaml
+# sed -i "s/ONLINEBOUTIQUE_NAMESPACE/onlineboutique/g" authorization-policies/for-namespace/kustomization.yaml
+# sed -i "s/ONLINEBOUTIQUE_NAMESPACE/onlineboutique/g;s/INGRESS_GATEWAY_NAMESPACE/asm-ingress/g;s/INGRESS_GATEWAY_NAME/asm-ingressgateway/g" authorization-policies/for-ingress-gateway/kustomization.yaml
+# sed -i "s/ONLINEBOUTIQUE_NAMESPACE/onlineboutique/g" sidecars/for-namespace/kustomization.yaml
```

Comment:
Same as before. Suggestion to remove the examples from inside the yaml files. Ideally I would put them on a README file at the root of every Kustomize file. 

Also having these here makes the string `ONLINEBOUTIQUE_NAMESPACE` get changed whenever the `sed` command is run to replace the strings

Reply:
Done.